### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by javidshah2** [guardianproject/orbot#1132]
**Why doesn't the new update come in the F-Droid repository?**

**Comment by javidshah2** [comment]
**Is there no one to answer my question?**

**Comment by n8fr8** [comment]
There is no good reason, other than a delay by people.

I will make sure our next Release Candidate is pushed there as soon as it is out.

**Comment by meenbeese** [comment]
> **I had a question, is Orbot version of F-droid de-Googled?**

No, it is not "de-Googled" because it was not "Googled" in the first place. It is the same binary APK file that you can get on GitHub or F-Droid, just distributed through a different distribution channel. I hope that answers your question, and if so, please consider closing this issue as we already have quite a few.

**Comment by meenbeese** [comment]
There are no differences between said versions and none of them use Firebase for push notifications.
